rasa_nlu/server.py

```diff
@@ -73,14 +74,30 @@ def parse_get():
             except InvalidModelError as e:
                 return jsonify({"error": "{}".format(e)}), 404
 
+    @rasa_nlu_app.route("/version", methods=['GET'])
+    @requires_auth
+    def version():
+        return jsonify({'version': __version__})
+
+    @rasa_nlu_app.route("/serverlog", methods=['GET'])
```

Comment:
I am afraid we can't do this one. The logs are process specific (e.g. when rasa server is started in a wsgi container with multiple processes, the different processes will use different log files).

As the others look good, I'd remove the log and we are good to go.